middleware/cache/cache.go

```diff
@@ -59,6 +59,8 @@ func New(config ...Config) fiber.Handler {
 	)
 	// Create manager to simplify storage operations ( see manager.go )
 	manager := newManager(cfg.Storage)
+	// Create indexed heap for tracking expirations ( see heap.go )
+	heap := &indexedHeap{}
```

Comment:
the allocations come from the fact that elements must be created here at runtime.

could you not create the array with fixed size already in the heapmap with the information about the max size ?

Reply:
Yes you are right